src/context/context-provider.spec.ts

```diff
@@ -199,6 +203,40 @@ describe('Context provider', () => {
     });
   });
 
+  it(
+    'should not redirect if disableRedirect and environment ID is required but the user is not in an environment',
+    async (done) => {
+      const expectedArgs: BBContextArgs = {
+        url: 'custom-url'
+      };
+
+      const invalidContextHandler = jasmine.createSpy('invalidContextHandler')
```

Comment:
Is this still needed based on your last changes? It doesn't appear to be referenced again after it's created.

Reply:
Shoot.  Nope.